src/components/comparison/comparison-side/comparison-side.tsx

```diff
@@ -496,25 +588,19 @@ export const ComparisonSide = ({
     setExamples((prevValues) =>
       prevValues.filter((example) => example.id !== id)
     );
-    setLayers((prevValues) =>
+    setActiveLayers((prevValues) =>
```

Comment:
Deleted layers don't disappear from the map

Reply:
Fixed